src/backgroundblurprocessor/BackgroundBlurOptions.ts

```diff
@@ -10,9 +10,19 @@ export default interface BackgroundBlurOptions {
   /** A {@link Logger|Logger} to which log output will be written. */
   logger?: Logger;
 
+  /** How often the video frame processor will report observed events. */
+  reportingPeriodMillis?: number;
+
   /** The amount of blur that will be applied to a video stream. */
   blurStrength?: number;
 
-  /** How often the video frame processor will report observed events. */
-  reportingPeriodMillis?: number;
+  /**
+   * The threshold CPU utilization to trigger skipping background filter updates which will reduce the amount of CPU
+   * used by background filtering. For example, If the reporting period is set to 1000 ms and 500 ms was dedicated to
+   * processing the background filter, then the CPU utilization for that reporting period is 50%. If
+   * {@link filterCPUUtilization} is set to 50 it will cause a `filterCPUUtilizationHigh` event to be fired from the
+   * `BackgroundBlurVideoFrameProcessorObserver`.
+   *
+   */
+  filterCPUUtilization?: number;
```

Comment:
What's the unit here? Percent? I.e 0-100 or 0-1.

Reply:
I'll update the name and comment to indicate that it is a percentage (0-100)